lib/Backend/InliningDecider.cpp

```diff
@@ -68,7 +68,7 @@ bool InliningDecider::InlineIntoInliner(Js::FunctionBody *const inliner) const
         return false;
     }
 
-    if (!inliner->GetAnyDynamicProfileInfo()->HasCallSiteInfo(inliner))
+    if (!inliner->IsJsBuiltInCode() && !inliner->GetAnyDynamicProfileInfo()->HasCallSiteInfo(inliner))
```

Comment:
Well seen ! Deprecated change.

---
In reply to: [122075745](https://github.com/Microsoft/ChakraCore/pull/3141#discussion_r122075745) [](ancestors = 122075745)